modules/machine-health-checks-creating.adoc

```diff
@@ -5,10 +5,14 @@
 
 :_content-type: PROCEDURE
 [id="machine-health-checks-creating_{context}"]
-= Creating a MachineHealthCheck resource
+= Creating a machine health check resource
 
-You can create a `MachineHealthCheck` resource for all `MachineSets` in your cluster.
-You should not create a `MachineHealthCheck` resource that targets control plane machines.
+You can create a `MachineHealthCheck` resource for machine sets in your cluster.
+
+[NOTE]
+====
+You can only apply a machine health check to control plane machines on clusters that use control plane machine sets.
+====
```

Comment:
Machine management > Deploying machine health checks > [Creating a MachineHealthCheck resource](https://53397--docspreview.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html#machine-health-checks-creating_deploying-machine-health-checks)

Reply:
Just add this bit I think